### PR TITLE
Update-converter-nue-to-enu

### PR DESCRIPTION
**Pull request by joachimhgg**
**Description**

Fix road conversion , improve general robustness, improve description

### CODE REVIEW

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
The script works really well in general but I saw some mistakes in the conversion for:
- [x] crossroads
- [x] plane and floor
- [x] stairs (in `spot.wbt`)
- [x] radiator, officeChair, cabinet, stairs (in `hall.wbt`)
- [ ] sofa (in `nao_room`)

The support for elevation_grid is missing.

I dont think everything need necessary to be fixed, it is something we can discuss...

I think it is important to explain in the wiki that it will only work for worlds that are in NUE. Worlds in ENU will be broken but I see no solution.

File: scripts/converter/convert_nue_to_enu_rub_to_flu.py [hunk 1]
```diff
@@ -310,11 +330,12 @@ def convert_nue_to_enu_world(filename, mode='all', objects_pi=[], objects_pi_2=[
 
     mode = 'all'  # specific, clean or all
     # possibility to use an argv, a list or a folder
-    filename_list = []  # example: 'projects/robots/robotcub/icub/worlds/icub_stand.wbt', change it by your .wbt or.proto
+    # example: 'projects/robots/robotcub/icub/worlds/icub_stand.wbt', change it by your .wbt or.proto
+    filename_list = ['projects/robots/softbank/nao/protos/Nao.proto']
```
[the remark this answers is not in the document]
Comment (joachimhgg) [reply scripts/converter/convert_nue_to_enu_rub_to_flu.py:334]: No, I am removing it.

**Comment by joachimhgg** [comment]
> The script works really well in general but I saw some mistakes in the conversion for:
> 
>     * [x]  crossroads
> 
>     * [x]  plane and floor
> 
>     * [x]  stairs (in `spot.wbt`)
> 
>     * [x]  radiator, officeChair, cabinet, stairs (in `hall.wbt`)
> 
>     * [x]  sofa (in `nao_room`)
> 
> 
> The support for elevation_grid is missing.
> 
> I dont think everything need necessary to be fixed, it is something we can discuss...
> 
> I think it is important to explain in the wiki that it will only work for worlds that are in NUE. Worlds in ENU will be broken but I see no solution.

I fixed the crossroad, viewpoint and added objects into the lists. For the `ElevationGrid`, it is too much time to do it so let's wait if there is a need for it, maybe we could share your script as well.

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**